app/api/helpers/mail.py

```diff
@@ -554,13 +561,6 @@ def send_email_to_attendees(order):
         order_view_url=order.site_view_link,
     )
 
-    event_date = convert_to_event_timezone(
-        order.event.starts_at, order.event.timezone, format='%d %B %Y'
-    )
-    event_time = convert_to_event_timezone(
-        order.event.starts_at, order.event.timezone, format='%H:%M (%Z%z)'
-    )
-
```

Comment:
Why remove these? Isn't it better to just change the parameters as this is being used repetitively. :thinking: